### DISCUSSION TITLE
cookiecutter for fastapi-users and Tortoise

### DISCUSSION THREAD

**Discussion by Tobi-De** [fastapi-users/fastapi-users#991] [Show and tell]
Hello, I hope everyone reading this is doing great

I'm building a cookiecutter for fastapi and tortoise-orm, it is not 100% complete but already usable, you can check it [here](https://github.com/Tobi-De/cookiecutter-fastapi)
The main features are :
- fastapi-users for authentication and user management.
- [Arq](https://github.com/samuelcolvin/arq) for task queue management
- [Tortoise-orm](https://github.com/tortoise/tortoise-orm) as the main database option (but other options will be available soon, I intend to add almost all databases supported by fastapi-users)

**Comment by frankie567** [comment]
Wow! That's really great @Tobi-De! Tons of interesting patterns here, great job!

**Reply by Tobi-De** [reply]
Thanks
